proposed/http-middleware/middleware-meta.md

```diff
@@ -270,75 +276,37 @@ additional information._
 
 [promises]: https://promisesaplus.com/
 
-### 5.2 Delegate Design
-
-The `DelegateInterface` defines a single method that accepts a request and
-returns a response. The delegate interface must be implemented by any middleware
-dispatcher that uses middleware implementing `MiddlewareInterface`.
-
-#### Why the term "delegate"?
-
-The term "delegate" means something designated to act for or represent another.
-In terms of middleware design, a delegate is called upon by middleware when the
-middleware is unable to process the request itself; the delegate then processes
-the request _for the original middleware_ in order to return a response.
-
-#### Why isn't the delegate a `callable`?
-
-Using an interface type hint improves runtime safety and IDE support.
-
-_See "discussion of FrameInterface" in [relevant links](#8-relevant-links) for
-additional information._
-
-#### Why doesn't delegate use `__invoke`?
+### 5.2 Handler Design
 
-Classes that define `__invoke` can be more loosely type hinted as
-`callable`, which results in less strict typing. This is generally undesirable,
-especially when the method uses strict typing.
+The `HandlerInterface` defines a single method that accepts a request and
+MUST return a response. The handler MAY delegate to another handler.
```

Comment:
The only reason I included that was to make it clear that a handler does not necessarily have to produce a response itself, eg: it can act as a delegate.